brands/amenity/cafe.json

```diff
@@ -832,6 +832,17 @@
       "name": "Muzz Buzz",
       "takeaway": "yes"
     }
+  },
+    "amenity/cafe|OLDTOWN White Coffee": {
+    "tags": {
+      "amenity": "cafe",
+      "brand": "OLDTOWN White Coffee",
+      "brand:wikidata": "Q7085308",
+      "brand:wikipedia": "en:Old Town White Coffee",
+      "cuisine": "malaysian;coffee_shop",
+      "name": "OLDTOWN White Coffee",
```

Comment:
Looking at their website, they don't seem to capitalize their name this way..

Reply:
I just follow thier website style.  https://www.oldtownmy.com/

Reply:
Let's just call it "OldTown White Coffee".  The spelling in the website doesn't seem to match how they sign it, or how most people would expect it spelled on a map pin.

![OldTown](https://user-images.githubusercontent.com/38784/83298020-b1162500-a1c1-11ea-9af3-78e2a3cdc675.jpg)

Reply:
okay, no problem